src/GitHub.App/ViewModels/GitHubPane/PullRequestDetailViewModel.cs

```diff
@@ -659,6 +676,26 @@ IObservable<Unit> DoPush(object unused)
                 });
         }
 
+        async Task DoSyncSubmodules(object unused)
+        {
+            usageTracker.IncrementCounter(x => x.NumberOfSyncSubmodules).Forget();
+
+            var progress = new CaptureProgress();
+            var complete = await pullRequestsService.SyncSubmodules(LocalRepository, progress);
+            if (!complete)
+            {
+                throw new ApplicationException(progress.ToString());
+            }
+        }
```

Comment:
Actually a low-impact way for now to show that something is happening during this method would be to do:

```
        async Task DoSyncSubmodules(object unused)
        {
            try
            {
                IsBusy = true;
                usageTracker.IncrementCounter(x => x.NumberOfSyncSubmodules).Forget();

                var progress = new CaptureProgress();
                var complete = await pullRequestsService.SyncSubmodules(LocalRepository, progress);
                if (!complete)
                {
                    throw new ApplicationException(progress.ToString());
                }
            }
            finally
            {
                IsBusy = false;
            }
        }
```

What do you think?